### PR TITLE
Misc improvements for maintenance effectiveness reporting

### PR DESCRIPTION
**Pull request by marco-c**

### CODE REVIEW

File: scripts/generate_landings_risk_report.py [hunk 1]
```diff
@@ -1787,22 +1787,20 @@ def calculate_maintenance_effectiveness(
...
+                return "ME: {}%, BurnDownTime: {} y, WeightedBurnDownTime: {} y".format(
                     round(me["ME"], 2),
                     round(me["BDTime"], 2),
                     round(me["WBDTime"], 2),
-                    round(me["Incoming vs total open"], 2),
-                    round(me["Closed vs total open"], 2),
                 )
 
-            maintenance_effectiveness_section = f"""<b>MAINTENANCE EFFECTIVENESS</b>
+            maintenance_effectiveness_section = f"""<b>[MAINTENANCE EFFECTIVENESS](https://docs.google.com/document/d/1y2dUDZI5U3xvY0jMY1LfIDARc5b_QB9mS2DV7MWrfa0/edit)</b>
```
Comment (suhaibmujahid) [inline scripts/generate_landings_risk_report.py:1796]: Nit:
```suggestion
            maintenance_effectiveness_section = f"""<b>[MAINTENANCE EFFECTIVENESS](https://docs.google.com/document/d/1y2dUDZI5U3xvY0jMY1LfIDARc5b_QB9mS2DV7MWrfa0)</b>
```

**Review by suhaibmujahid — APPROVED** [review]
Thank you. LGTM.

File: scripts/generate_landings_risk_report.py (hunk 1 above)
Comment (marco-c) [reply scripts/generate_landings_risk_report.py]: Done in b1ed7f8.

File: bugbug/bugzilla.py [hunk 2]
```diff
@@ -443,8 +443,8 @@ def calculate_maintenance_effectiveness_indicator(
     }
 
     logger.info(
-        "Calculating maintenance effectiveness indicator for the %s team from %s to %s",
-        team,
+        "Calculating maintenance effectiveness indicator for the %s teams from %s to %s",
+        ", ".join(teams),
```
Comment (suhaibmujahid) [inline bugbug/bugzilla.py:447]: Nit: we could use "and": https://github.com/suhaibmujahid/bugbot/blob/a929257d756715933ec2672ac63da38ed66eeeaf/bugbot/utils.py#L181-L186

**Review by suhaibmujahid — APPROVED** [review]

**Pull request merged**